runtime-common/core/core-context.h

```diff
@@ -4,10 +4,13 @@
 
 #pragma once
 
-#include <cstddef>
+#include <array>
 #include <cstdint>
+#include <cstring>
+#include <optional>
 
 #include "common/mixin/not_copyable.h"
+#include "common/wrappers/string_view.h"
```

Comment:
Why this include is added here?

Reply:
Previously there was string view instead of array. I'll remove it